docs-source/spring/content/development/cli/_index.md

```diff
@@ -560,3 +560,179 @@ The following is an example development workflow using the CLI:
          oractl:>config list --service-name myserv --service-profile obaas --service-label 0.1 --property-key ktest2
          400 : "Couldn't find any property for submitted query."
          ```
+8. Use the `GraalVM Compile Commands` to:
+* upload a **.jar** file to the Oracle Backend for Spring Boot and Microservices and its GraalVM compiler service;
+* start a compilation of your microservice to produce an executable native file **.exec**;
+* get last logs available regarding a compilation in progress or terminated;
+* download the **.exec** file to deploy on the backend
+* purge files remaining after a compilation on remote GraalVM compiler service.
+
+The GraalVM Compile Commands are the following:
+
+```cmd
+oractl:>help 
+   
+GraalVM Compile Commands
+       compile-download: Download compiled executable file
+       compile: Compile a service with GraalVM
+       compile-purge: Delete a launched job
+       compile-logs: Compilation progress
+```
+
+   1. Use the `compile` command to upload and automatically start compilation using the following syntax:
+
+```cmd
+   oractl:>help compile
+       NAME
+       compile - Compile a service with GraalVM
+
+       SYNOPSIS
+              compile [--artifact-path String] --help 
+
+       OPTIONS
+              --artifact-path String
+              Service jar to compile location
+              [Mandatory]
+
+              --help or -h 
+              help for compile
+              [Optional]
+```
+
+   Since the compilation of a .jar file via the tool `native-image` does not support cross-compilation, so it must be on the same kind of platform where the application will run, this service guarantees a compilation in the same Operating System and CPU type where the service it will be executed on the Kubernetes cluster.
```

Comment:
Change to:

Because the compilation of a .jar file using the ... cross-compilation, it must ... will run. This service ... operating system ... the service will be ...